src/agg_bookend.c

```diff
@@ -395,12 +391,12 @@ ts_first_sfunc(PG_FUNCTION_ARGS)
 }
 
 /* last(internal internal_state, anyelement value, "any" comparison_element) */
-Datum
-ts_last_sfunc(PG_FUNCTION_ARGS)
+Datum ts_last_sfunc(PG_FUNCTION_ARGS)
```

Comment:
The `Datum` should be on its own line here.

Reply:
I don't think clang supports this

Reply:
I don't think clang supports this